### DISCUSSION TITLE
Unexpected "Risk Profile" microCOVID Numbers

### DISCUSSION THREAD

**Discussion by nh905** [microCOVID/microCOVID#994] [Q&A] [answered]
My assumption is that microCOVIDs shown for "An average person in your area" is calculated using the location-based data on the left side of the screen.  Yesterday, my wife and I were seeing different risk profile microCOVIDs for the same location.  For Israel, one of us was seeing an average risk of 10,000 while the other saw 20,000.  Similarly, for Canada/Ontario/Niagara, one was seeing 400 and the other 900.  Refreshing the screen and the browser cache did not change the numbers.  I verified that the inputs for the locations were the same, and neither of us was overriding location-based data.  Unfortunately I was busy with an unrelated problem and did not collect all the required information.  

I think it may be related to using a premade vs. custom scenario.  In the past, we would start with a premade scenario and tweak it.  My wife has recently being using custom scenarios since for a while it was not obvious how to access the list of premade scenarios.  When I changed from a custom to a premade scenario, I noticed that the "An average person in your area" microCOVIDs  changed.  However, various attempts at recreating the lower figure using different custom scenarios were not successful.

So far, the problem has not reoccurred.  However, the only reason that we noticed the discrepancy is that I had mentioned a possible concern with a planned trip to Niagara region in mid September.  There was also an article on BBC News about the resurgence of covid-19 in Israel and we were interested in seeing how its microcovid.org numbers compared with Florida.  If the risk profiles changed between successive calculations on our own laptops, we would have put it down to the underlying data having changed.

I wish I had more details, but decided to post this in case other people are seeing something similar. 

Regards, Norbert

**Comment by nh905** [comment, marked as answer]
The mystery is solved.  Changing "Their vaccine: Are they fully vaccinated?" through Yes/No/I don't know changes the microCOVIDs score of the average person.  For Canada/Ontario/York the values are 400, 2000, and 900.  I see the logic, but it is different from how other factors are  handled as adjustments to an initial risk assessment.  It is also somewhat inconsistent with "An average person ..." - I can only determine if the people around me are vaccinated (or not) if I am dealing with specific people that I know.  In my case, it also created confusion because I assumed (based on past application behavior) that the "Average person in your area" was tied to the incidence of covid-19 in my area.

Regards, Norbert

**Reply by blanchardjeremy** [reply]
Glad to know you figured it out. Tagging @jy2wong as she's working on usability in this area.

**Reply by blanchardjeremy** [reply]
And yes, it is tied to the incidence of COVID 19 in your area. You might be able to know that the average person is vaccinated if you're at a venue that requires vaccination. Or a party where everyone is vaccinated, but you don't know anything else about their behavior. 

Hope this helps!

**Reply by nh905** [reply]
I understand the logic. but  the "Their vaccine:" control works differently from the others that either calculate a situation risk or apply an adjustment, leading to confusion when my wife and I were seeing different Ontario microCOVIDs scores.   Including a note might have helped us narrow down the issue quicker.

I can still use the "Average person" microCOVIDs score as a quick way to track baseline community risk and also compare risk between communities. but have to remember to set "Their vaccine:" to "I don't know".
